### PR TITLE
feat: added year of release and deprecation #2

### PR DESCRIPTION
**Pull request by sreejitStrob**
- https://en.wikipedia.org/wiki/Ajax_(programming) Ajax first appeared in March of 1999.
- https://en.wikipedia.org/wiki/Angular_(web_framework) Released on 14 September 2016

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,132 +3,132 @@
...
-- [Anime.js](https://animejs.com) - Library
-- [Ant Design](https://ant.design) - UI Library
-- [Apollo](https://www.apollographql.com) - Library
-- [Axios](https://axios-http.com) - Library
+- [Ajax](https://en.wikipedia.org/wiki/Ajax_(programming)) - Programming Technique (1999)
+- [Angular](https://angular.io) - Framework (2016)
+- [Anime.js](https://animejs.com) - Library 
+- [Ant Design](https://ant.design) - UI Library 
+- [Apollo](https://www.apollographql.com) - Library 
+- [Axios](https://axios-http.com) - Library
```
Comment (paradite) [inline README.md:11]: Remove the empty space when there is no date info

File: scripts/generateReadme.js [hunk 2]
```diff
@@ -8,12 +8,20 @@ content += '[List by categories](categories.md)\n\n';
...
       : term.name;
+
+    if (term.year_created && term.year_deprecated) {
+      dateInfo = `(${term.year_created} - ${term.term.year_deprecated})`
+    } else if (term.year_created) {
+      dateInfo = `(${term.year_created})`
+    }
     return `- ${nameWithLink}${
-      term.type ? ` - ${term.type}` : ''
+      term.type ? ` - ${term.type} ${dateInfo}`  : ''
```
Comment (paradite) [inline scripts/generateReadme.js:24]: Update the script for generating categories too.

**Review by paradite — CHANGES_REQUESTED** [review]

**Comment by paradite** [comment]
Please update https://github.com/paradite/frontend-encyclopedia/blob/main/scripts/generateCategories.js as well

**Comment by sreejitStrob** [comment]
> Please update https://github.com/paradite/frontend-encyclopedia/blob/main/scripts/generateCategories.js as well

I have added the changes for categories too.

**Review by paradite — APPROVED** [review]

**Comment by paradite** [comment]
Thank you. Merged.

**Comment by sreejitStrob** [comment]
> Thank you. Merged.

You welcome.

**Pull request merged**